docs/event_details/screen_view.md

```diff
@@ -0,0 +1,23 @@
+# screen_view
```

Comment:
**question**: I don't see a difference between `screen_opened` and `screen_view`. I think we can use just `screen_opened`.

**chore**: `screen_view` is special event automatically collected by FirebaseAnalytics: https://support.google.com/analytics/answer/9234069?hl=en. We shouldn't reuse this event, because it will make our life harder destingushing those two events.